src/index.ts

```diff
@@ -738,6 +739,117 @@ export class MerklePatriciaTree {
     return {value, proof};
   }
 
+  getBulk(keys: Buffer[]): Witness[] {
+    const ptree = new patriciaTree();
+    for (const key of keys) {
+      ptree.insert(Buffer.from(key).toString("hex"));
```

Comment:
Don't convert Buffers into strings. This adds a significant amount of overhead.

Reply:
Okay. Not sure if I can get away with this if I'm using the patricia-tree library. I'll look into it.

Reply:
I'm not sure we should be using this library. Can't you just use a map?